Documentation/admission-controller-usage.md

```diff
@@ -0,0 +1,80 @@
+# Admission Controller
+
+## Requirements
+
+1. Webhook server which will validate the requests
+2. TLS Secrets
+3. ValidatingWebhookConfig/MutatingWebhookConfig
+4. RBAC Components 
+
+The first component is integrated with Rook itself. Last three requirements are given in deploy.sh script file. 
+
+## Quick Start 
+
+For deploying admission controllers with rook we have the script file deploy.sh (examples/kubernetes/ceph/admission-controller). 
+
+Run the following commands
+```
+kubectl create -f common.yaml
+./deploy.sh
+```
+Now that the secrets have been deployed, we can deploy the operator and create a cluster using the following commands
+```
+kubectl create -f operator.yaml
+```
+
+At this point all the above requirements have been fulfilled and the webhook can start intercepting requests for rook resources.
+
+## ValidatingWebhookConfig
```

Comment:
The admin doesn't need this section in the docs. This is an implementation detail.

Reply:
Removed